sentry.client.config.ts

```diff
@@ -9,12 +9,11 @@
 import * as Sentry from "@sentry/nextjs";
 
 Sentry.init({
+  environment: process.env.NEXT_PUBLIC_APP_ENV,
   dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
 
   // Adjust this value in production, or use tracesSampler for greater control
-  tracesSampleRate: ["development", "heroku"].includes(process.env.NODE_ENV)
-    ? 1.0
-    : 0.1,
+  tracesSampleRate: ["development"].includes(process.env.NODE_ENV) ? 1.0 : 0.1,
```

Comment:
very ignorant and lazy Q: is "NODE_ENV" special? Why wouldn't this need a "NEXT_PUBLIC_" prefix too?